dex/candles/candles.go

```diff
@@ -130,8 +130,8 @@ func (c *Cache) Delta(since time.Time) (changePct float64, vol uint64) {
 		if candle.StartStamp <= cutoff && candle.StartRate != 0 {
 			// Interpret the point linearly between the start and end stamps
 			cut := float64(cutoff-candle.StartStamp) / float64(candle.EndStamp-candle.StartStamp)
-			rateDelta := candle.EndRate - candle.StartRate
-			r := candle.StartRate + uint64(cut*float64(rateDelta))
+			rateDelta := float64(candle.EndRate) - float64(candle.StartRate)
```

Comment:
Oh signed integers.

Trying to gauge impact.  There going to be candles in the epoch_reports with crazy spot rate changes, and there are going to be crazy candles from the `/candles` endpoint, but on restart will all that correct itself when `AddMarketSource` >  `LoadEpochStats` runs?

Reply:
Can you also update `TestDelta` with a `Candle` where `EndRate` is less than `StartRate`?

Reply:
> Oh unsigned integers.
> 
> Trying to gauge impact. There going to be candles in the epoch_reports with crazy spot rate changes, and there are going to be crazy candles from the `/candles` endpoint, but on restart will all that correct itself when `AddMarketSource` > `LoadEpochStats` runs?

That's right. Nothing is wrong in the DB, so everything should correct itself on restart.